### PR TITLE
JSF 2.2 FAT: JSFSimpleHtmlUnit updtes for Faces 4.0

### PR DESCRIPTION
**Pull request by pnicolucci**
for #22369

### CODE REVIEW

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
#libby #build

**Comment by pnicolucci** [comment]
Java 17 build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_0qHK8ECqEe23scQ7k9ObvQ

**Comment by pnicolucci** [comment]
Java 17 build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_KJY6EEEmEe23scQ7k9ObvQ

File: dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22SimpleHTML.war/src/com/ibm/ws/jsf22/fat/simple/bean/faces40/CompositeComponentTestBean.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2015, 2019 IBM Corporation and others.
```
Comment (volosied) [inline dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22SimpleHTML.war/src/com/ibm/ws/jsf22/fat/simple/bean/faces40/CompositeComponentTestBean.java]: Copyrights should be updated here (along with the other files in new packages), I believe?

**Review by volosied — APPROVED** [review]
Looks good.

**Comment by pnicolucci** [comment]
#libby #build

**Pull request merged**